NTupleMaker/src/Synch17GenTree.cc

```diff
@@ -57,19 +57,29 @@ void Synch17GenTree::ReadInit(TTree *tree)
    fCurrent = -1;
    fChain->SetMakeClass(1);
 
-   fChain->SetBranchAddress("Higgs_pt", &Higgs_pt, &b_Higgs_pt);
-   fChain->SetBranchAddress("Higgs_eta", &Higgs_eta, &b_Higgs_eta);
-   fChain->SetBranchAddress("Higgs_phi", &Higgs_phi, &b_Higgs_phi);
-   fChain->SetBranchAddress("Higgs_mass", &Higgs_mass, &b_Higgs_mass);
+   fChain->SetBranchAddress("gen_Higgs_pt", &Higgs_pt, &b_Higgs_pt);
```

Comment:
Just for consistency, I would suggest to remove the "gen_" prefix in these variables